examples/update_bids_datasets.py

```diff
@@ -0,0 +1,147 @@
+"""
```

Comment:
when an example does not start with plot_ it is not executed and rendered in the doc.

can you avoid this?

note that caching the mne somato data on circle ci is what mne does.

Reply:
Sorry I don't follow. Not as familiar with sphinx if that's the issue here. None of the other examples start with `plot_`, and the example is rendered here?

https://3407-89170358-gh.circle-artifacts.com/0/dev/auto_examples/update_bids_datasets.html#sphx-glr-auto-examples-update-bids-datasets-py

Reply:
you're right ! see https://github.com/mne-tools/mne-bids/blob/master/doc/conf.py#L145

we execute every example even if it does not start with plot_

Reply:
--> time to rename the examples in `mne-python` :)

Reply:
yeah this was a conscious choice because most of our examples don't "plot" anything.